modules/ROOT/pages/mule-expression-language-date-and-time-functions.adoc

```diff
@@ -402,7 +402,7 @@ Equivalent to http://docs.oracle.com/javase/8/docs/api/java/util/Calendar.html#a
 |===
 |Function |Description |Return Type
 |`withTimeZone(timezone);` a|
-Changes the current DateTime to match a defined timezone. Effectively changing the dateTime and the timezone of the instance.
+Changes the timezone of the instance. Effectively changing only the timezone of the instance.
```

Comment:
Is it needed to clarify that "Effectively changes?" Wouldn't be ok to just say: 

* Changes only the timezone of the instance. 
* Changes the timezone of the instance, and then changes the current `DateTime` to match the updated timezone.

Reply:
Updated.